### ISSUE TITLE
Spinning wheel when importing a file

### ISSUE THREAD

**Issue by alabella19** [CDCgov/PoSE#2]
I installed PoSE on my Windows machine (Windows 10 x64) with Matlab runtime R2020a v. 9.8.0.1323502. 

When I go to import a file I hear a ding and then a spinning wheel over the majority of the window. I can attempt to import another file or exit the program. I have left the spinning wheel for ~15 minutes and nothing changes. 

Any assistance in getting PoSE running would be greatly appreciated. This software is exactly what I have been looking for. 
Best,
Abigail

**Comment by CDC-Kun** [comment]
Abigail,

Could you please open a Windows command prompt and launch the app from there?  Then, try running the steps again.  If there are any errors, these will appear in the Windows command prompt.

•	Launch Windows Command Prompt app (From Windows start menu, type cmd)
•	Navigate to the location where the application is installed:
>> cd "C:\Program Files\PoSE\application"
•	Type PoSE.exe to launch the app
>> PoSE.exe

As tries to import the file,  any errors logged to the Windows command prompt? 

Thank you,

**Comment by alabella19** [comment]
I don't know if this makes a difference but it is installed under `C:\Program Files\BasemlTranslatorTool\application` and the application is called BasemlTranslatorTool.exe. 

(I installed the program from PoSE-1.2beta.exe downloaded yesterday)

I launched BasemlTranslatorTool.exe, the GUI opened up, and I tried to import the test file. 

There were no errors logged on the Windows command prompt. 

Thank you so much for your prompt response! 
Abigail

**Comment by CDC-Kun** [comment]
Sorry for the confusion.  

Yes. It should be "BasemlTranslatorTool.exe".   We will change the name later, but that's the current program name.  Glad that it works and let us know if further assistance is needed.

Thanks.

Kun

**Comment by alabella19** [comment]
I'm still getting the spinning wheel even though no errors showed up. Sorry for the confusion.

**Comment by CDC-Kun** [comment]
The installation folder should not make a difference.  Could you please provide a screenshot of the app after you import the test file?  Would it also be possible for you to share the file?  

Here is my email:  kzhao@cdc.gov. We can try to reproduce this.  

We are also happy to debug this over a webex with you if that’s easier/possible.  Thank you.

**Comment by miyuxuezhe** [comment]
Hi, I am trying to use PoSE to visualize the output of PAML. When I install it on different computers (win10 home version and server version), there is no problem. But when I loaded the example data(baseml_rst-win.txt and codeml_rst-win.txt), the software interface kept loading, and I waited overnight without visualizing the example data. I am running CMD with administrator privileges and executing commands as you said. The software did not report an error but still did not visualize the example data correctly. Kindly ask for your help, thank you

**Comment by CDC-Kun** [comment]
Gustin Zhang,

Could you please confirm: 

1. you are using "PoSE-1.2-beta.exe".
2. baseml_rst and codeml_rst were loaded into their corresponding button correctly.  

Thank you.

Kun

**Comment by miyuxuezhe** [comment]
> Gustin Zhang,
> 
> Could you please confirm:
> 
> 1. you are using "PoSE-1.2-beta.exe".
> 2. baseml_rst and codeml_rst were loaded into their corresponding button correctly.
> 
> Thank you.
> 
> Kun

Yes, I have previously downloaded matlab runtime 2020a from the official website and installed it correctly in the default path (C:\Program Files\MATLAB\MATLAB Runtime\v98), then downloaded the PoSE software from github, double-clicked to install PoSE-1.2-beta.exe, and used After installing the software in the default installation path and the default matlab runtime path (C:\Program Files\MATLAB\MATLAB Runtime\v98), open BasemlTranslatorTool.exe, File -> import and import the sample data (baseml_rst-win. txt or codeml_rst-win.txt). The software reports no errors, but it still cannot visualize the sample data correctly.

**Comment by alabella19** [comment]
Gustin,
   Sorry for the delay in the response. I figured out the error was
associated with unix coding in the test files downloaded with the program.
Once I re-coded the files in windows format they worked fine.
Abbe

Dr. Abigail Leavitt LaBella
Postdoctoral Scholar, Vanderbilt University
PhD University Program in Genetics and Genomics

abigail.labella@gmail.com

**Comment by miyuxuezhe** [comment]
Yes, there is no error when the example file format is changed to the windows encoding format.

Thanks

**Comment by MauriAndresMU1313** [comment]
Hi!
I have the same issue.
The program never shows de results of example data. 
How fix the problem? 
For example, @alabella19 re-coded the data. How I can do this?
I´m don´t have sufficient experience in programming.
Can you suggest how you fix the problem?

**Comment by CDC-Kun** [comment]
> Yes, there is no error when the example file format is changed to the windows encoding format.
> 
> Thanks

Glad it works.  Thank you.
